src/components/NavBar/index.tsx

```diff
@@ -1,69 +1,66 @@
 import React from 'react';
 import { Colors, Styles } from '../../constants';
-import { MainMenu, IconProps } from './MainMenu';
+import { MainMenu, IconProps, IconStyle } from './MainMenu';
 import { SbNavLink, SbNavlinkProps } from '../SbNavLink';
 import { SearchBar } from '../SearchBar';
 import { HelpCircle, Menu } from 'react-feather';
+import MediaQuery from 'react-responsive';
+import { Link } from 'react-router-dom';
 
 /*tslint:disable: no-require-imports no-var-requires */
-const sbLogo: string = require('@sbac/sbac-ui-kit/src/images/SmarterBalanced-Logo.png') as string;
+const sbLogo = require('@sbac/sbac-ui-kit/src/images/SmarterBalanced-Logo.png') as string;
 
 export interface NavBarProps {
-  links?: SbNavlinkProps[];
   siteName: string;
   mainContentId: string;
 }
 
-export const Logo = (): JSX.Element => (
-  <div className="nav-titleGroup-item">
-    <a
-      rel="noopener noreferrer"
-      target="_blank"
-      href="http://www.smarterbalanced.org/"
-      title="Smarter Balanced Home"
-    >
-      <img alt="Smarter Balanced Logo" src={sbLogo} />
-    </a>
+export const HeaderLogo: React.SFC = () => (
```

Comment:
This isn't used in a header, it's used in the NavBar. Calling it a `HeaderLogo` is a bit of misnomer.

Reply:
I think NavBarLogo is a bit of a misnomer as well because typically the website's logo goes in the `<header>` tag not the `<nav>` one. Looks like @peterhen0 just put it there to reduce the complexity of the component